tools/scylla-nodetool.cc

```diff
@@ -402,7 +408,11 @@ void enablegossip_operation(scylla_rest_client& client, const bpo::variables_map
 }
 
 void flush_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
-    const auto [keyspace, tables] = parse_keyspace_and_tables(client, vm);
+    const auto [keyspace, tables] = parse_keyspace_and_tables(client, vm, false);
```

Comment:
I wanted to prevent such a param to `parse_keyspace_and_tables()`, because the caller can already easily check `vm.count("keyspace")` itself. No need to add noop code to the above method.

Reply:
So if the keyspace option is required the other caller should verify that?

Reply:
Since the `keyspace_required` flag is set to `true` by default, existing call sites need not change.

Reply:
kept in next version.
let's resume discussion after I push it.

Reply:
Ok, I have no strong opinion on this, it is fine this way too.